src/orange/OrangeData.hh

```diff
@@ -213,14 +211,15 @@ struct OrangeParamsData
     Items<SimpleUnitRecord> simple_unit;
 
     // Low-level storage
-    Items<SurfaceId> surface_ids;
-    Items<VolumeId> volume_ids;
+    Items<LocalSurfaceId> local_surface_ids;
     Items<RealId> real_ids;
     Items<logic_int> logic_ints;
     Items<real_type> reals;
     Items<SurfaceType> surface_types;
     Items<Connectivity> connectivities;
-    VolumeItems<VolumeRecord> volume_records;
+    Items<LocalVolumeId> local_volume_ids;
```

Comment:
Not sure why this moved down here? It doesn't matter but please make sure the order of the data and the assignment/bool/etc are the same.